src/main/resources/i18n/messages.properties

```diff
@@ -1701,6 +1701,21 @@ announcement.user.readon=Read by you
 announcement.markRead=Mark as Read
 announcement.markUnread=Mark as Unread
 
+AnnouncementTable.title=Title
+
+CreateNewAnnouncement.title=Add Announcement
+CreateNewAnnouncement.okBtn=Create Announcement
+
+EditAnnouncement.title=Edit Announcement
+EditAnnouncement.okBtn=Update Announcemnet
+
+DeleteAnnouncement.title=Delete this announcement?
+DeleteAnnouncement.ok=Delete
+
+MarkdownEditor.write=Write
+MarkdownEditor.paste=Paste
+MarkdownEditor.Preview=Preview
```

Comment:
Change .Preview to .preview

Reply:
Removed in a138363d1e6f63de7e8d0deff1146219a8904c38